aggregator/src/test/java/com/redhat/cloud/notifications/DailyEmailAggregationJobTest.java

```diff
@@ -44,6 +45,7 @@
     void setUp() {
         helpers.purgeEmailAggregations();
         System.clearProperty("notifications.aggregator.email.subscription.periodic.cron.enabled");
+        testee.registry = new SimpleMeterRegistry();
```

Comment:
Note for later:
Injected fields should not be instantiated with `new` in a `QuarkusTest`, they should be mocked instead.

Reply:
Yes, this test is also a bit misleading as it's testing with mocks, as asll as integration testing without mocks.
I think in the long term it's better to split the test.